src/tests/test_solve_postfix_equation.py

```diff
@@ -84,8 +84,6 @@
     ("0 sin 1 + 0 cos +", 2),
     ("43 sin 69.69 + 1 cos +", 69.3985275632),
     ("100 sin 3 / 33.852 cos -", 0.59248644973),
-    ("0 _ 1 + 0 cos +", 2),
-    ("984526.123654 _ 100 _ + 1 sin - 0 cos *", 1001.3914269),
```

Comment:
sqrt by měl být ve tvaru v postfixu "x(základ) y(exponent) _", ty to máš infixově "x(základ) _ y(exponent)"